frontend/app/src/components/EvmNativeTokenBreakdown.vue

```diff
@@ -44,31 +49,52 @@ const breakdowns = computed(() => {
   });
 });
 
-const { t } = useI18n();
+const { tc } = useI18n();
 const { currencySymbol } = storeToRefs(useGeneralSettingsStore());
 
 const tableHeaders = computed<DataTableHeader[]>(() => {
-  return [
+  const headers: DataTableHeader[] = [
     {
-      text: t('common.location').toString(),
+      text: tc('common.location'),
       value: 'location',
       align: 'center',
-      width: '120px'
+      class: 'text-no-wrap'
     },
     {
-      text: t('common.amount').toString(),
+      text: tc('common.amount'),
       value: 'balance.amount',
-      align: 'end'
+      align: 'end',
+      width: '60%'
     },
     {
-      text: t('asset_locations.header.value', {
+      text: tc('asset_locations.header.value', 0, {
         symbol: get(currencySymbol) ?? CURRENCY_USD
-      }).toString(),
+      }),
       value: 'balance.usdValue',
       align: 'end'
     }
   ];
+
+  if (get(showPercentage)) {
+    headers.push({
+      text: tc('asset_locations.header.percentage'),
+      value: 'percentage',
+      align: 'end',
+      class: 'text-no-wrap',
+      sortable: false
+    });
+  }
+
+  return headers;
 });
+
+const percentage = (value: BigNumber) => {
+  const netWorth = get(total);
```

Comment:
This is not the `netWorth` anymore right it is the total value of the asset.
Can we rename the variable just to avoid confusion?